Automattic/jetpack
# VideoPress: Use remote image for processing thumbnail

We were having problems with subdirectories and how the image path was formed in the VideoPress API on first upload of the video. 

This (Jetpack) code is only used on subsequent reloads, but pointing to the (new) remote image instead of relying on local will enable us to only update the image in one place in the future.  

To Test: 
- Activate VideoPress and upload a video from `/wp-admin/post-new.php`
- Ensure the thumbnail image appears correctly. 
- Quickly reload the page and see that the same thumbnail with the same img src appears correctly.  

related: 
- r148618-wpcom
- r148615-wpcom
